ideascube/configuration/errors.py

```diff
@@ -0,0 +1,21 @@
+class InvalidConfigurationValue(Exception):
```

Comment:
Python use exception to control a normal execution.
For example, the StopIteration to stop a for loop.

If the Exception is a real error, the exception is named with Error (IOError, ValueError, ...)